spec/rspec/core_spec.rb

```diff
@@ -325,4 +339,3 @@ def reporter
     expect(status.exitstatus).to eq(0)
   end
 end
```

Comment:
Please revert unrelated changes. We'll have to deal with merging a growing list of changes from 3.x branch to 4.0, that removed many things, and would love to keep the volume of changes here as low as possible.
We'll update RuboCop, and adjust/enforce the rules later on. Not to say that we need to fix that `rubocop` works differently locally and on CI.